x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/expression/function/FunctionRegistry.java

```diff
@@ -310,6 +313,26 @@ public boolean functionExists(String functionName) {
         return def(function, builder, false, aliases);
     }
 
+    /**
+     * Build a {@linkplain FunctionDefinition} for multi-arg function that
+     * is not aware of time zone and does not support {@code DISTINCT}.
+     */
+    @SuppressWarnings("overloads") // These are ambiguous if you aren't using ctor references but we always do
+    static <T extends Function> FunctionDefinition def(Class<T> function,
+            MultiFunctionBuilder<T> ctorRef, String... aliases) {
+        FunctionBuilder builder = (location, children, distinct, tz) -> {
+            if (distinct) {
+                throw new IllegalArgumentException("does not support DISTINCT yet it was specified");
+            }
+            return ctorRef.build(location, children);
+        };
+        return def(function, builder, false, aliases);
+    }
+
```

Comment:
Really really minor: can you remove the empty line or add it to the other places between the `def` variant and the iface?

Reply:
Sure, I've added it to the rest (the source wasn't consistent).